src/server/actions/wrappers.ts

```diff
@@ -80,8 +81,8 @@ export function adminAction<S extends Schema, F extends WrappedFunc<S>>(func: F,
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const wrappedFunction = async (arg: z.infer<S>): Promise<any> => {
         const store = localStorageContext.getStore()
-
-        if (store?.orgSlug !== 'safeinsights') {
+        if (store?.orgSlug !== 'safe-insights') {
```

Comment:
lets add the slug as an exported variable somewhere, maybe in server/config ?  That way hopefully we don't typo it again like I did here 😭

Reply:
will do! One moment...